pkg/etl/doc.txt

```diff
@@ -0,0 +1,11 @@
+/**
+ * @defgroup pkg_etl Embedded Template Library
+ * @ingroup  pkg
```

Comment:
Since this package is purely for C++ we could think about adding it to cpp group instead of pkg group. What do you think?

Reply:
I think it's better to put it to C++, I changed it because no one has expressed a different opinion yet.